internal/lib/constants.go

```diff
@@ -95,6 +95,10 @@ const (
 	DefaultSEGroup                             = "Default-Group"
 	GatewayFinalizer                           = "gateway.ako.vmware.com"
 	ClusterStatusCacheKey                      = "cluster-runtime"
+
+	//Specifies command used in namespace event handler
+	Add    = "ADD"
+	Delete = "DELETE"
```

Comment:
Can we rename the above constants to something which tells us what it does? For e.g.,
```
NsFilterAdd = "ADD"
NsFilterDelete = "DELETE"
```